nginx.master.conf

```diff
@@ -158,21 +158,7 @@ http {
         }
 
         location ~ ^/(slave|agent)/(?<agentid>[0-9a-zA-Z-]+)(?<url>.*)$ {
-            access_by_lua 'auth.validate_jwt_or_exit()';
-            set $agentaddr '';
-            set $agentport '';
-
-            more_clear_input_headers Accept-Encoding;
-            rewrite ^/(slave|agent)/[0-9a-zA-Z-]+/.*$ $url break;
-            rewrite_by_lua_file conf/master/agent.lua;
-
-            include common/proxy-headers.conf;
-            include common/server-sent-events.conf;
-
-            # Supports chunked encoding and stream for debugging API
-            proxy_request_buffering off;
-
-            proxy_pass http://$agentaddr:$agentport;
+						return 301 $scheme://system/v1/agent/$agentid/mesos/$uri;
```

Comment:
I was on the line between doing a rewrite and a 301. We can do a `rewrite ... permanent;` instead if that's what is desired. In the talks we had yesterday, we talked about `redirect` which I interpreted as a 301. 

I'll fix the spacing, thanks.